mods/species/vox/_vox.dm

```diff
@@ -9,6 +9,10 @@
 	credits_crew_names = list("THE VOX")
 	credits_topics = list("VOX RITUAL DUELS", "NECK MARKINGS", "ANCIENT SUPERCOMPUTERS")
 
+/decl/modpack/vox/post_initialize()
+	. = ..()
+	global.item_damage_test_examptions |= /obj/item/gun/energy/darkmatter
```

Comment:
Not entirely sure. But it would runtime during unit testing when created on its own. Couldn't figure why from glancing over its really simple definition. Might have to do with the extension?